react/tic-tac-toe/src/app/screens/Login/index.js

```diff
@@ -3,25 +3,31 @@ import { connect } from 'react-redux';
 import { SubmissionError } from 'redux-form';
 import PropTypes from 'prop-types';
 
-import { fetchUsers, sendToken } from '../../../redux/users/actions';
+import { fetchUsers, sendToken, loggedUser } from '../../../redux/users/actions';
 
 import LoginForm from './components/LoginForm';
 
 class LoginFormContainer extends Component {
-  componentDidMount() {
-    const { getUsers, dispatchToken, history } = this.props;
+  async componentDidMount() {
+    const { getUsers, dispatchToken, history, selectedUser } = this.props;
+    await getUsers();
```

Comment:
remove await and async and you shouldnt get the users list to  login 👀, just send user and passwrod the the api